pgctl/cli.py

```diff
@@ -238,6 +238,11 @@ def __change_state(self, state):
                     for service in self.all_services
                 )
 
+            # Run the playground-wide "post-start" hook (if it exists) upon every `pgctl start'
+            if state is Start:
+                if all(service.state['state'] == 'ready' for service in self.services):
+                    self._run_playground_wide_hook('post-start')
```

Comment:
The main concern is to avoid race condition as "start-msg" is designed to since "post-start" is treated as an extension of "start-msg" -- "post-start" serves for almost the same purpose as "start-msg" except when it is triggered.